x-pack/plugins/security_solution/public/management/pages/blocklist/view/components/blocklist_form.tsx

```diff
@@ -351,16 +376,17 @@ export const BlockListForm = memo(
           ? [GLOBAL_ARTIFACT_TAG]
           : change.selected.map((policy) => `${BY_POLICY_ARTIFACT_TAG_PREFIX}${policy.id}`);
 
+        const nextItem = { ...item, tags };
+
         setSelectedPolicies(change.selected);
+        validateValues(nextItem);
```

Comment:
This fixes an error where the form was enabled for submit when changing the assignment section even there was no name or values already set.